_events/2019-09-13-bacon-night.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Bacon Night
+featured: /assets/images/events/bacon_night.jpg
+location: Our house
+address: >-
+  114 N. Orchard St.
+  Madison, WI 53715
+---
+
+Join us for our semi-annual night for celebrating a cultural delicacy, bacon! Starting at 6 and going through the evening, we'll keep the skillets hot and the strips coming. Come any time and stay as long as you want. It's guaranteed to be a packed house and a great chance to meet the members.
+<!-- More -->
+Who doesn't love bacon‽ Well, a few religions, diets, and life-styles choices. Don't worry, we'll have other food and drink available! If the smell's still too strong for ya, we understand. We'll make sure to air out the kitchen before you [visit us at one of our other events]({{site.baseurl}}/events). ;)
```

Comment:
2nd sentence grammar is a little off, I would just get rid of it. Do we really need the last two sentences?